mongo/database.go

```diff
@@ -161,8 +165,13 @@ func (db *Database) processRunCommand(ctx context.Context, cmd interface{},
 		Database(db.name).Deployment(db.client.deployment).ReadConcern(db.readConcern).Crypt(db.client.crypt), sess, nil
 }
 
-// RunCommand runs a command on the database. A user can supply a custom
-// context to this method, or nil to default to context.Background().
+// RunCommand executes the given command against the database.
+//
+// The runCommand parameter should be a document for the command to be executed. It cannot be nil.
+// This must be an order-preserving type such as bson.D. Map types such as bson.M are not valid.
+// If the command document contains a session ID or any transaction-specific fields, the behavior is undefined.
```

Comment:
I added this line (and the corresponding line for `RunCommandCursor`) because this is mentioned in the transactions spec.